src/transformers/models/t5/modeling_t5.py

```diff
@@ -833,7 +831,6 @@ def forward(
         # Model parallel
         if self.model_parallel:
             torch.cuda.set_device(self.first_device)
-            self.embed_tokens = self.embed_tokens.to(self.first_device)
```

Comment:
this has already been called in `parallelize` - did I miss that perhaps it may have changed since `parallelize` or would it be better to call it here rather than in `parallelize`?